src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -151,20 +154,20 @@
     <Compile Include="System\Data\SqlTypes\SqlXml.cs" />
     <Compile Include="$(CommonPath)\System\NotImplemented.cs" />
     <Compile Include="$(CommonPath)\System\Data\Locale\Locale.cs">
-        <Link>System\Data\Locale\Locale.cs</Link>
+      <Link>System\Data\Locale\Locale.cs</Link>
```

Comment:
I thought it would be better to have the FieldNameLookup.cs at CommonPath so that it lives with the base class that it needs. But it makes sense to keep it in SqlClient because it doesn't have to used by System.Data.Common anymore. 

As a follow up of this PR, I need to 
1. Modify the System.Data.Common to expose DbEnumerator 
2. Adoption of the new version of System.Data.Common in SqlClient. 
3. Remove up DbEnumerator and related classes from SqlClient. I will take care of moving the FieldNameLookup to SqlClient in this step.